### ISSUE TITLE
[Bug]: Cloud save not functioning properly

### ISSUE THREAD

**Issue by Kapv4** [MelvorIdle/melvoridle.github.io#2797]
### Describe the bug

When I Cloud save on PC and use Cloud save on phone and vice versa my progress seems to have 2 different instances of saves. 
For example: i train astrology on PC and when I log onto mobile its still training thieving. And then i check astrology on mobile and when I log onto pc it's still training thieving. 
So basically my progress has been reset 2-3 times since thrones of herald launched. 
Before thrones of herald it was working flawlessly. 

### Save Link

https://melvoridle.com/save/bZZ3zmRZ

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] Mobile App (iOS)
- [X] Mobile App (Android)
- [X] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.1.0

### Game Subversion

?4598

### Are you using any mods?

No

**Comment by ekuland** [comment]
You're definitely tapping 'show cloud saves' at the top and loading from those?

**Comment by aiydee** [comment]
I've noticed something similar, but what is worthwhile is that the PC version drops sync to the account when the PC goes into a screensave.  (I'm guessing network connection turns off).  When this happened before, it'd resync with my user when it came back online and happy days. 
But now, it reports 'online' but the Account logged in is blank.  
I found as a dirty work around that I had to log out, log back in using local save.  Sync to cloud then exit to be able to sync the phone up.
